app/helpers/era_av_helper.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017, University of Alberta Libraries.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+#   under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#   specific language governing permissions and limitations under the License.
+# ---  END LICENSE_HEADER BLOCK  ---
+
+module EraAvHelper
+  def ualib_logo_link
+    link_to("", "https://library.ualberta.ca", :class => 'logo')
+  end
+
+  def era_av_logo_link
+    image =
+      image_tag("ERA-A+V-beta-logo.png",
+                alt: "University of Alberta ERA Audio and Video"\
+                "- #{t(:release_label)} #{Avalon::VERSION}")
+    return image if current_page?(root_path)
```

Comment:
I personally find it annoying when a link such as this one just takes the browser to the same page. So yeah, personal taste.

Regarding the helper, the U of A logo link appeared twice in the same page called with the same args (to handle a couple of resonsive-rendering scenarios), so I thought putting it in a helper would DRY it up.

In the case of the ERA logo (with the logic to handle home vs non-home linking), the helper cleans up the view template. I'd rather see logic in helpers/presenters than in erb templates (where reasonable).